Cesium3DTiles/include/Cesium3DTiles/RasterOverlayTileProvider.h

```diff
@@ -198,75 +186,25 @@ class CESIUM3DTILES_API RasterOverlayTileProvider {
     return this->_projection;
   }
 
-  /**
-   * @brief Returns the {@link CesiumGeometry::QuadtreeTilingScheme} of this
-   * instance.
-   */
-  const CesiumGeometry::QuadtreeTilingScheme& getTilingScheme() const noexcept {
-    return this->_tilingScheme;
-  }
-
-  /**
-   * @brief Returns the coverage {@link CesiumGeometry::Rectangle} of this
-   * instance.
-   */
-  const CesiumGeometry::Rectangle& getCoverageRectangle() const noexcept {
-    return this->_coverageRectangle;
-  }
-
-  /**
-   * @brief Returns the minimum tile level of this instance.
-   */
-  uint32_t getMinimumLevel() const noexcept { return this->_minimumLevel; }
-
-  /**
-   * @brief Returns the maximum tile level of this instance.
-   */
-  uint32_t getMaximumLevel() const noexcept { return this->_maximumLevel; }
-
-  /**
-   * @brief Returns the image width of this instance.
-   */
-  uint32_t getWidth() const noexcept { return this->_imageWidth; }
-
-  /**
-   * @brief Returns the image height of this instance.
-   */
-  uint32_t getHeight() const noexcept { return this->_imageHeight; }
-
   /**
    * @brief Returns the {@link RasterOverlayTile} with the given ID, creating it
    * if necessary.
    *
-   * @param id The {@link CesiumGeometry::QuadtreeTileID} of the tile to obtain.
+   * @param id The {@link TileID} of the tile to obtain.
    * @return The tile.
    */
   CesiumUtility::IntrusivePointer<RasterOverlayTile>
-  getTile(const CesiumGeometry::QuadtreeTileID& id);
+  getTile(const TileID& id, const CesiumGeometry::Rectangle& imageryRectangle);
```

Comment:
I don't think this method is needed in the base class. This implementation should be moved to the `QuadtreeRasterOverlayTileProvider`, and `RasterizedPolygonsOverlay` can just create a new tile every time without the awkward use of an incremented int as an ID.

Reply:
Same with `getTileWithoutCreating` below.